plugins/GSdx/GSRendererHW.cpp

```diff
@@ -35,12 +35,12 @@ GSRendererHW::GSRendererHW(GSTextureCache* tc)
 	m_userhacks_align_sprite_X = !!theApp.GetConfig("UserHacks_align_sprite_X", 0) && !!theApp.GetConfig("UserHacks", 0);
 	m_userhacks_round_sprite_offset = !!theApp.GetConfig("UserHacks", 0) ? theApp.GetConfig("UserHacks_round_sprite_offset", 0) : 0;
 
-	if (m_upscale_multiplier == 1 && !m_nativeres) { //Custom
+	if (m_upscale_multiplier == 7) { //Custom Resolution
 		m_width = theApp.GetConfig("resx", m_width);
 		m_height = theApp.GetConfig("resy", m_height);
 	}
 
-	if (m_upscale_multiplier == 1) {
+	if (m_upscale_multiplier == 7) { //Custom Resolution
```

Comment:
actually, m_upscale_multiplier == 1 means check for custom resolution formerly. either way, fixed it.